Makefile

```diff
@@ -84,6 +90,76 @@ else
 	@echo Update complete
 endif
 
+## zkgroup
+build-zkgroup:
+	@echo "Found cargo with version $(CARGO_VERSION)"
+	@echo "Found go with version $(GO_VERSION)"
+	@echo "Found go with version $(GIT_VERSION)"
+ifeq ($(UNAME_S), Linux)
```

Comment:
The way I see it, the only thing differing the two sections is the last `mv`, where either libzkgroup_linux_amd64.so or libzkgroup_linux_arm64.so is used.

As we already know the arch with the $(UNAME_HARDWARE_PLATTFORM) variable, could we not simplify the whole section like this?

```
build-zkgroup:
	@echo "get zkgroup $(PLATFORM)"
	go get -d github.com/nanu-c/zkgroup
	&& git submodule update \
	&& cd lib/zkgroup \
	&& cargo build --release --verbose
	mv libzkgroup.so libzkgroup_linux_$(UNAME_HARDWARE_PLATTFORM).so
```